master/internal/hpimportance/manager.go

```diff
@@ -59,23 +71,39 @@ type stateRecord struct {
 }
 
 type manager struct {
+	config   HPImportanceConfig
 	db       *db.PgDB
 	state    map[int]stateRecord
 	pool     pool.ActorPool
 	disabled bool
 }
 
 // NewManager initializes the master actor (of which there should only be one instance running).
-func NewManager(db *db.PgDB, system *actor.System, config HPImportanceConfig) actor.Actor {
+func NewManager(db *db.PgDB, system *actor.System, config HPImportanceConfig, masterRoot string,
+) (actor.Actor, error) {
+	// growforest should either be installed in PATH (when running from source) or package with the
+	// master (when running from binary packages).
+	growforest := path.Join(masterRoot, growforestBin)
+	_, err := os.Stat(growforest)
+	if os.IsNotExist(err) {
+		growforest = growforestBin
+	}
+
+	err = os.MkdirAll(workingDir, 0700)
```

Comment:
talked offline, on restart (so here) we should wipe everything under this dir, to clean of stuff that got left from a previous run.